history/characters/wc_easteregg_characters.txt

```diff
@@ -457,4 +457,276 @@ easteregg13={ #Shrek reference
+	}
+	# level 2 warrior
+	662.2.19 = { death = yes }
+}
+
+# Tauren mercenary, therefore a landless playable character
+easteregg17 = {
+	name = "Jubai"
+	#dna = jubai_dna
+	culture = tauren
+	religion = earth_mother_worship
+	disallow_random_traits = yes
+	#Stats
+	diplomacy = 5
+	martial = 4
+	stewardship = 3
+	intrigue = 3
+	learning = 6
+	#Personality
+	trait = brave
+	trait = forgiving
+	trait = honest
+	#Education
+	trait = education_learning_3
+	#Other
+	trait = adventurer
+	trait = magic_good_1
+	506.3.10 = { 
+		birth = yes
+		trait = creature_tauren
+	}
+	583.1.1 = {
+		give_nickname = nick_the_ancient
+	}
+	# level 3 druid
+	641.6.10 = { death = yes }
+}
+
+# Dwarven officer of Ironforge
+easteregg18 = {
+	name = "Serphentos"
+	dna = serphentos_dna
+	culture = bronzebeard
+	religion = khazism
+	disallow_random_traits = yes
+	#Stats
+	diplomacy = 2
+	martial = 7
+	stewardship = 5
+	intrigue = 5
+	learning = 3
+	#Personality
+	trait = vengeful
+	trait = honest
+	trait = brave
+	#Education
+	trait = education_martial_3
+	#Other
+	trait = physique_good_2
+	trait = lifestyle_blademaster
+	377.8.24 = { 
+		birth = yes
+		trait = creature_dwarf
+	}
+	511.7.15 = {
+		employer = 17001 # Magni [4200]
+		give_council_position = councillor_marshal
+	}
+	# level 3 warrior
+	627.1.19 = { death = yes }
+}
+
+# Sentinel captain of the Alliance
+easteregg19 = {
+	name = "Monóri"
+	dna = monori_dna
+	female = yes
+	dynasty = easteregg_dyn11 # Silverleaf
+	culture = night_elf
+	religion = kaldorei_religion
+	disallow_random_traits = yes
+	#Stats
+	diplomacy = 6
+	martial = 7
+	stewardship = 4
+	intrigue = 3
+	learning = 3
+	#Personality
+	trait = compassionate
+	trait = generous
+	trait = diligent
+	#Education
+	trait = education_martial_3
+	#Other
+	trait = beauty_good_3
+	trait = lifestyle_hunter
+	22.10.21 = { 
+		birth = yes
+		trait = creature_night_elf
+		effect = { add_character_flag = no_headgear }
+	}
+	507.1.1 = {
+		employer = 340008 # Malfurion [128003]
+		give_council_position = councillor_marshal
+	}
+	610.1.21 = { # Killed and risen as undead by Arthas
+		trait = being_undead
+		religion = death_god
+	}
+	# level 3 huntress, later dark ranger
+	610.8.18 = { death_reason = death_battle }	# Killed in Northrend
+}
+
+# Necromancer on the service of the Lich King who had infiltrated Stormwind
+easteregg20 = {
+	name = "Thainor"
+	dna = thainor_dna
+	culture = lordaeronian
+	religion = holy_light
+	disallow_random_traits = yes
+	#Stats
+	diplomacy = 3
+	martial = 2
+	stewardship = 3
+	intrigue = 6
+	learning = 8
+	#Personality
+	trait = deceitful
+	trait = ambitious
+	trait = patient
+	#Education
+	trait = education_learning_3
+	#Other
+	trait = intellect_good_1
+	trait = magic_good_2
+	558.6.3 = { 
+		birth = yes
+		trait = creature_human
+	}
+	593.1.1={
+		employer = 10014		# Ner'zhul [2353]
+		religion = death_god
+	}
+	# level 3 necromantic
+	610.1.10 = { death_reason = death_battle }	# Killed in Ironforge by Serphentos
```

Comment:
If he was killed by serphentos shouldn't that be reflected in the code?
Like:
```
610.1.10 = { 
   death = {
        death_reason = death_battle 
        killer = easteregg18
    }
}
```

Reply:
fixed